packages/components/organisms/f-checkout/src/store/checkout.module.js

```diff
@@ -356,9 +356,19 @@ export default {
 
             if (isAddressInLocalStorage) {
                 const storedAddress = addressService.getAddressFromLocalStorage(false);
+
                 if (addressService.doesAddressInStorageAndFormMatch(storedAddress, state.address)) {
                     addressCoords = [storedAddress.Field2, storedAddress.Field1];
                     commit(UPDATE_GEO_LOCATION, addressCoords);
+                } else if (window.localStorage) {
```

Comment:
Apologies if I don't see it clearly, but Line 357 checks `isAddressInLocalStorage` and on this line here we check for `window.localStorage` is there some opportunity to optimise this? Use the `addressService` for example.

Reply:
Fixed.